x-pack/plugins/integration_assistant/common/api/model/common_attributes.gen.ts

```diff
@@ -85,6 +97,14 @@ export const SamplesFormat = z.object({
    * For some formats, specifies whether the samples can be multiline.
    */
   multiline: z.boolean().optional(),
+  /**
+   * For CSV format, specifies whether the samples have a header row.
```

Comment:
`header` is not particular for `CSV` format alone. We are using this field to represent presence of header in other log formats including `structured` , `unstructured`

Reply:
Fair enough. Though I wonder if the other header should be called something else, like `syslog: boolean`?

Reply:
The comment you added should help.